src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -301,6 +305,18 @@ const TreeItem: React.FC<TreeItemProps> = props => {
         ]
       : []),
     {key: 'divider-4', type: 'divider'},
+    ...(git.repo
```

Comment:
We should still check in here if the `remoteURL` contains the `https://github.com`

Reply:
Maybe check only for `github` ? Probably there are multiple domains that work..

Reply:
As mentioned before, what happens if your GitLab repo has a name like `my-github-test` ?

Reply:
Hey I thought if its not a github then repo will return falsy and I tried with gitlab it was not showing link but let me be sure 👍 I will check in detail and come back

Reply:
Hmm directly works on gitlab too since its a git repo too lol I am a little confused, should I put open on gitlab if its on gitlab? or not show anything? @devcatalin @topliceanurazvan

Reply:
updated to check only github for now as mentioned on task 👍